test/e2e/performanceprofile/functests/1_performance/hugepages.go

```diff
@@ -148,7 +148,7 @@ var _ = Describe("[performance]Hugepages", Ordered, func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			cmd2 := []string{"/bin/bash", "-c", "tmux new -d 'LD_PRELOAD=libhugetlbfs.so HUGETLB_MORECORE=yes top -b > /dev/null'"}
-			_, err = pods.ExecCommandOnPod(testclient.K8sClient, testpod, cmd2)
+			_, err = pods.ExecCommandOnPod(testclient.K8sClient, testpod, "", cmd2)
```

Comment:
this works. An alternative option could have been to add a variant function which accepts a container name and reimplement the existing ExecCommandOnPod (with the current signature) on top of it. But we can keep this approach.

Reply:
Yes this is an option as well.